Cargo.lock

```diff
@@ -382,6 +418,21 @@ dependencies = [
  "bitflags",
 ]
 
+[[package]]
+name = "ring"
+version = "0.16.20"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3053cf52e236a3ed746dfc745aa9cacf1b791d846bdaf412f60a8d7d6e17c8fc"
+dependencies = [
+ "cc",
+ "libc",
+ "once_cell",
+ "spin",
+ "untrusted",
+ "web-sys",
```

Comment:
I don't get why they have web-sys here.

Reply:
So it seems like deno_lockfile depends on ring, which in turn depends on web-sys (but this isn't included in the compilation, just included in the lock file). https://github.com/briansmith/ring/pull/904 is talking about it